cylc/flow/task_job_mgr.py

```diff
@@ -1299,7 +1320,9 @@ def get_job_conf(
                 itask.platform['job runner command template']
             ),
             'dependencies': itask.state.get_resolved_dependencies(),
-            'directives': rtconfig['directives'],
+            'directives': self.default_dict_merger(
+                itask.platform['directives'], rtconfig['directives']
+            ),
```

Comment:
Is it not possible to do?
```suggestion
            'directives': {
                **itask.platform['directives'], **rtconfig['directives']
            },
```